docs/theming.md

```diff
@@ -185,9 +185,174 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+This migration guide covers the typical migration path from Themer usage to Theming extensions.
+
+Theming extensions are discussed in detail above. Themers are discussed in the [Themers](#themers)
+section. Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
+
+Some Theming extensions are still in Beta. To add a Beta Theming extension to your project you will
+need to follow the instructions in the [Beta components](../contributing/beta_components.md)
+documentation. Non-Beta Theming extensions can be added to your project like any other target
+provided in the MDC repo.
```

Comment:
Nit: "the MDC repo" sounds rather informal/unclear. "...can be added to your project like everything else in MDC-iOS."